Java/ds/CheckBST.java

```diff
@@ -0,0 +1,227 @@
+package neoAlgo;
+
+/*This is the solution to create a Binary Tree from user inputs taken level wise and to check if that binary tree is a Binary Search tree.This solution works for duplicate input values of nodes as well.
```

Comment:
Reduce the length of the line. Compact sentences will help the readers to better understand the Code